tutorials/matching_fof.ipynb

```diff
@@ -424,6 +424,41 @@
     "plt.xlim(lims);\n",
     "plt.ylim(lims);"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ellipticity comparison plot is quite surprising. \n",
+    "Not only they don't match, but it seems the ellipticity in the object catalog is higher when compared to that in the extragalactic catalog. \n",
+    "If anything, the PSF should smear the ellipticity (making the value smaller) in the object catalog. \n",
+    "\n",
+    "We should however, remind ourselves the definition of ellipticity used in these catalogs. \n",
+    "For the extragalactic catalog, ellipticity is defined as $(1-q)/(1+q)$\n",
+    "(see the [SCHEMA](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md#schema-for-extragalatic-catalogs)). \n",
+    "On the other hand, for the object catalog, we are using the HSM ellipticity that is defined as $(1-q^2)/(1+q^2)$\n",
+    "(see e.g., Eq. 8 of [Mandelbaum et al. 2006](https://arxiv.org/abs/astro-ph/0511164)).\n",
+    "\n",
+    "With some math, we can find the conversion between the two definitions $e_{\\rm HSM~def} = \\frac{2e_{\\rm EGC~def}}{1+e_{\\rm EGC~def}^2}$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# compare the ellipticity (smartly)\n",
+    "conversion = lambda e: 2*e / (1.0+e*e)\n",
+    "plt.figure(figsize=(5,5));\n",
+    "plt.scatter(conversion(matched_gals['ellipticity_true']), matched_gals['shape_hsm_regauss_etot'], s=1);\n",
```

Comment:
This is a super minor point (and isn't about a new aspect of the notebook), but I thought that `ellipticity_true` is before the application of lensing shear, while the observed shape is after, so it's not totally apples-to-apples for that reason as well?  Perhaps this is worth noting in the discussion of the shapes in the catalog, so people realize that this distinction could be relevant for some purposes, unless another tutorial already covers the distinction between unlensed and lensed quantities in the extragalactic catalogs.